sklearn/naive_bayes.py

```diff
@@ -299,7 +319,20 @@ def partial_fit(self, X, y, classes=None, sample_weight=None):
         self : object
             Returns self.
         """
-        return self._partial_fit(X, y, classes, _refit=False,
+        # deprecation warning if classes passed:
+        if classes is not None:
+            warnings.warn("The classes argument will be removed in version"
+                          "0.21. A new classes argument has been added as a"
+                          "class parameter since version 0.19 which is"
+                          "recommended to be used now.", DeprecationWarning)
+
+        # Raise error if classes set in both initialization and in partial_fit
+        if (self.classes is not None) & (classes is not None):
```

Comment:
I think we prefer "and" instead of the bitwise and, but it shouldn't matter here?